scripts/visualize.py

```diff
@@ -43,59 +42,35 @@ def main(_):
     if random_seed is not None:
         torch.manual_seed(random_seed)
 
-    dataset = meshnets.utils.datasets.FromDiskGeometricDataset(FLAGS.data_dir)
-    size_dataset = len(dataset)
-    num_training = int(FLAGS.train_split * size_dataset)
-    _, validation_dataset = torch.utils.data.random_split(
-        dataset, [num_training, size_dataset - num_training])
-
-    wrapper = meshnets.utils.model_loading.load_model_from_mlflow(
-        FLAGS.tracking_uri, FLAGS.run_id, FLAGS.checkpoint)
-
-    sample_idx = np.random.choice(validation_dataset.indices)
-
-    mesh_path = dataset.get_mesh_path(sample_idx)
-    graph_path = dataset.get_graph_path(sample_idx)
+    dataset = datasets.load_dataset(
```

Comment:
Given that you need to do all this preparation, wouldn't be better to create a separate method for this?

Reply:
Yes